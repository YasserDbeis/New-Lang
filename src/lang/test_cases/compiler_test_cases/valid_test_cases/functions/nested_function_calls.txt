func main()
{
    a();
}

func a()
{
    print("entering a");
    b();
    print("leaving a");
}

func b()
{
    print("entering b");
    c();
    print("leaving b");
}

func c()
{
    print("entering c");
    print("leaving c");
}