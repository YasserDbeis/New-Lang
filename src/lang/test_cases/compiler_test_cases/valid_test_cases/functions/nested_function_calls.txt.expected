entering a
entering b
entering c
leaving c
leaving b
leaving a